contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -566,6 +559,28 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         return _getNextMessageID(chainID);
     }
 
+    /**
+     * @dev Returns the number of receipts that have been received from the given chain ID.
```

Comment:
For comment we can do something like `See {ITeleporterMessenger-getReceiptQueueSize`